cpp/ql/src/change-notes/released/0.0.10.md

```diff
@@ -1,4 +1,5 @@
----
-category: deprecated
----
+## 0.0.10
+
+### Deprecated Queries
+
```

Comment:
This change note appears to be in the wrong folder. It sounds like these are library changes, and not query changes.

Reply:
Indeed. This should have been in lib.

Reply:
Except that this "library" is located in `src` not `lib`

Reply:
Ok, in that case we might want to change the section header to something like `Deprecated classes` or so.

Reply:
Sounds sensible.

The [changed file](https://github.com/github/codeql/blob/main/cpp/ql/src/external/CodeDuplication.qll) should probably live in `lib`, but since everything [here](https://github.com/github/codeql/tree/main/cpp/ql/src/external) is deprecated it doesn't seem to be worth moving at this point.

Reply:
That would also be a breaking change which I don't think we could actually do anyway.

Reply:
Note that if you update the wording in this file, you should probably also apply the same changes to `cpp/ql/src/CHANGELOG.md`

Reply:
How should I go about updating this? Fix this manually on the branch to be merged (now https://github.com/github/codeql/pull/8233), or differently?

Reply:
Yes, I think so. I also pushed an update to the C# change logs.